src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private static ColorSupplier supplier = new ColorSupplier();
```

Comment:
lets use ColorSupplier in FigureSupplier class, and pass random color as constructor parameter

Reply:
Move ColoSupplier from Figure class to FigureSupplier class.

Reply:
And lets remove it from `Figure` class